XCTest.xcodeproj/project.pbxproj

```diff
@@ -409,13 +407,22 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				INFOPLIST_FILE = Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				MACOSX_DEPLOYMENT_TARGET = 10.11;
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
-				SDKROOT = macosx;
```

Comment:
I did the following:

- move SUPPORTED_PLATFORMS from project level to 'XCTest' target.
- move deployment targets  from project level to 'XCTest' target. Because there're relevant only for 'XCTest' target.
- revert SDKROOT to macosx for project level.